python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -431,6 +431,9 @@ def aten〇repeat〡shape(self: List[int], repeats: List[int]) -> List[int]:
     for i in range(tensor_dim):
         out.append(self[i] * repeats[i + leading_rank])
     return out
+    
+def aten〇repeat_interleave〇Tensor〡shape(repeats: List[int], output_size: Optional[int] = None) -> List[int]:
+    return repeats
```

Comment:
Isn't the output size somehow depending on the sum of entries in `repeats`? At least this is how I read https://pytorch.org/docs/stable/generated/torch.repeat_interleave.html#torch.repeat_interleave